inspirehep/modules/workflows/utils.py

```diff
@@ -176,3 +177,41 @@ def download_file_to_workflow(workflow, name, url):
         if req.status_code == 200:
             workflow.files[name] = req.raw
             return workflow.files[name]
+
+
+def store_original_record_for_source(record_id, source, data):
+    """Store the original record for a given source.
+
+    Given a previously matched ``record_id`` (i.e. its UUID), the ``source``
```

Comment:
If the `record_id` is in fact a `uuid` it should be called `record_uuid`, no? Otherwise I would expect to be able to put a `control_number` there.